pkg/common/router/trie/trie.go

```diff
@@ -18,12 +18,13 @@
 package trie
 
 import (
-	"github.com/pkg/errors"
+	"strings"
 )
 
 import (
 	"github.com/apache/dubbo-go-pixiu/pkg/common/util/stringutil"
 	"github.com/apache/dubbo-go-pixiu/pkg/logger"
+	"github.com/pkg/errors"
```

Comment:
please split third party import to another import block,like below.

import (
	"github.com/apache/dubbo-go-pixiu/pkg/common/util/stringutil"
	"github.com/apache/dubbo-go-pixiu/pkg/logger"
)

import (
       "github.com/pkg/errors""
)